src/index.html

```diff
@@ -3,11 +3,85 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+           user-scalable=no,
+           initial-scale=1.0,
+           maximum-scale=1.0,
+           minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="Product Image"
+          class="card__logo"
+        >
+      </div>
+
+      <div class="card__title-block">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br>5K Monoblock (MNED2UA/A)
+        </p>
+      </div>
+
+      <div class="card__name">
+        <p class="card__product-code">Product code: 195434</p>
+      </div>
+
+      <div class="card__ranking">
+        <div class="stars">
```

Comment:
Add the modifier classes here
```suggestion
        <div class="stars stars--4">
```